tests/jest/unit/app/modules/import-export/assets/js/admin.test.js

```diff
@@ -0,0 +1,121 @@
+describe( 'Import Export Admin Test', () => {
+	window.elementorAppConfig = {
+		'import-export': {
+			lastImportedSession: {
+				kit_title: 'Kit Title',
+			},
+		},
+	};
+
+	require( 'elementor/app/modules/import-export/assets/js/admin.js' );
+
+	it( 'should check if "kit deleted" dialog is not shown without cache', () => {
+		// Arrange
+		window.elementorCommon = {
+			dialogsManager: {
+				createWidget: jest.fn( () => {
+					return {
+						show: jest.fn( () => {} ),
+					};
```

Comment:
you don't need to mock "show" if you don't use it

Reply:
It is called in the class